DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/README.md

```diff
@@ -218,24 +220,24 @@ In this tutorial, we will show how to use the `kernel_args_restrict` attribute f
 
 Locate `report.html` in the `kernel_args_restrict_report.prj/reports/` directory.
 
-Navigate to the *Loop Analysis* report (*Throughput Analysis* > *Loop Analysis*). In the *Loop List pane*, you should see two kernels: one is the kernel without the attribute applied (*KernelArgsNoRestrict*) and the other with the attribute applied (*KernelArgsRestrict*). Each kernel has a single for-loop, which appears in the *Loop List* pane. Click the loop under each kernel to see how the compiler optimized it.
+Navigate to the *Loop Analysis* report (*Throughput Analysis* > *Loop Analysis*). In the *Loop List pane*, you should see four kernels as described in the [Tutorial Code Description](#tutorial-code-description) section of this README. Each kernel has a single for-loop, which appears in the *Loop List* pane. Click the loop under each kernel to see how the compiler optimized it.
 
-Compare the loop initiation interval (II) between the two kernels. Notice that the loop in the *KernelArgsNoRestrict* kernel has a large estimated II, while the loop in the *KernelArgsRestrict* kernel has an estimated II of ~1. These IIs are estimates because the latency of global memory accesses varies with runtime conditions.
+Compare the loop initiation interval (II) between the kernels with or without `[[intel::kernel_args_restrict]]` attribute. Notice that the loop in the *KernelArgsNoRestrict_Lambda* and *KernelArgsNoRestrict_Functor* kernels has a large estimated II, while the loop in the *KernelArgsRestrict_Lambda* and *KernelArgsRestrict_Functor* kernels has an estimated II of ~1. These IIs are estimates because the latency of global memory accesses varies with runtime conditions.
```

Comment:
```suggestion
Compare the loop initiation interval (II) between the kernels with or without `[[intel::kernel_args_restrict]]` attribute. Notice that the loops in the *KernelArgsNoRestrict_Lambda* and *KernelArgsNoRestrict_Functor* kernels have large estimated IIs, while the loops in the *KernelArgsRestrict_Lambda* and *KernelArgsRestrict_Functor* kernels have an estimated II of 1. These IIs are estimates because the latency of global memory accesses varies with runtime conditions.
```